src/feedbacks/loading/spinner/PSpinner.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+    <svg class="p-spinner"
+         :width="size"
+         :height="size" viewBox="0 0 32 32"
+         fill="none" xmlns="http://www.w3.org/2000/svg"
+    >
+        <g>
+            <circle class="spinner-bg"
+                    cx="16" cy="16" r="14"
+                    :stroke="bgColor"
+                    stroke-opacity="0.3"
+                    stroke-width="9%"
+            />
+            <circle class="spinner"
+                    cx="16" cy="16" r="14"
+                    :stroke="spinnerColor"
+                    stroke-width="9%"
+                    stroke-linecap="round" stroke-dasharray="32 200"
+            />
+        </g>
+    </svg>
+</template>
+
+<script lang="ts">
+import {
+    defineComponent,
+} from 'vue';
+
+
+interface Props {
+    size?: string;
+    color?: string;
+}
+
+export default defineComponent<Props>({
+    name: 'PSpinner',
+    components: {
+    },
+    props: {
+        size: {
+            type: String,
+            default: '2rem',
+        },
+        spinnerColor: {
+            type: String,
+            default: '#898995',
+        },
+        bgColor: {
+            type: String,
+            default: '#A7A9B2',
+        },
```

Comment:
What if we all use color tokens for defaultValue?

There is an additional utility function called getColors.
Using this function, it takes our color token as input and returns it as a hash value.
Of course, a hash value such as '#fff' is returned as a hash value as it is.
Refer to the badge component for better understanding.

It would be better if the two props could be modified to receive color values flexibly :)

---

defaultValue에는 모두 컬러 토큰을 사용하면 어떨까요?

추가로 getColors라는 유틸 함수가 있습니다.
해당 함수를 이용하면, 우리의 컬러 토큰을 입력 받아 해시값으로 리턴해줍니다.
물론 '#fff'와 같은 해시값도 그대로 다시 해시값으로 리턴해줍니다.
badge 컴포넌트를 참고하시면 이해가 빠르실 거에요.

두 개의 props가 컬러 값을 유연하게 받을 수 있도록 수정해주면 더 좋을 것 같습니다 :)

---

### with color token

![image](https://user-images.githubusercontent.com/26986739/196150077-83debbba-ad3f-4375-9885-6a61f70e90f3.png)

### with hash value

![image](https://user-images.githubusercontent.com/26986739/196150321-61b8ac81-71c0-4ef1-a68c-8b5f12fca9ff.png)

Reply:
The component spec has changed and I have modified it in a different way. Thanks for the comment.

---

컴포넌트 스펙이 변경되어 다른 방식으로 수정했습니다. 코멘트 감사합니다.